app/src/main/java/io/gnosis/safe/ErrorHandler.kt

```diff
@@ -91,12 +93,23 @@ sealed class Error(
     object Error1110 : Error(1110, null, R.string.error_client_safe_name_invalid_reason, R.string.error_client_safe_name_invalid_fix)
     object Error1111 : Error(1111, null, R.string.error_client_key_already_imported_reason, R.string.error_client_key_already_imported_fix)
 
-    object ErrorUdUnsupportedDomain: Error(6357, null, R.string.error_client_UD_invalid_domain_reason, R.string.error_client_UD_invalid_domain_fix)
-    object ErrorUdUnregistered: Error(6358, null, R.string.error_client_UD_name_not_registered_reason, R.string.error_client_UD_name_not_registered_fix)
-    object ErrorUdRecordNotFound: Error(6359, null, R.string.error_client_UD_record_not_found_reason, R.string.error_client_UD_record_not_found_fix)
-    object ErrorUdUnspecifiedResolver: Error(6360, null, R.string.error_client_UD_domain_not_configured_reason, R.string.error_client_UD_domain_not_configured_fix)
-    object ErrorUdBlockhainDown: Error(6361, null, R.string.error_client_UD_blockchain_provider_is_not_accessible_reason, R.string.error_client_UD_blockchain_provider_is_not_accessible_fix)
-    object ErrorUdUnknownCurrency: Error(6357, null, R.string.error_client_UD_currency_not_found_reason, R.string.error_client_UD_currency_not_found_fix)
+    object ErrorUdUnsupportedDomain : Error(6357, null, R.string.error_client_UD_invalid_domain_reason, R.string.error_client_UD_invalid_domain_fix)
```

Comment:
Error 6357 and all following errors are not defined in error spec  https://docs.google.com/document/d/1foQ9DSMgO2iN-6u4JsJIDJWB7mUHX-D8ApZWvx6oKAo/edit?ts=5ff5e055#. Could you please add them? Also we should stick to our naming scheme.